sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_users_services_first_seen_v1/query.sql

```diff
@@ -1,136 +1,93 @@
-WITH base AS (
+WITH fxa_events AS (
   SELECT
-    * REPLACE (
-      IF(service IS NULL AND event_type = 'fxa_activity - cert_signed', 'sync', service) AS service
-    )
+    timestamp,
+    user_id,
+    IF(service IS NULL AND event_type = 'fxa_activity - cert_signed', 'sync', service) AS service,
+    os_name,
+    flow_id,
+    event_type,
+    os_version,
+    country,
+    language,
+    utm_term,
+    utm_source,
+    utm_medium,
+    utm_campaign,
+    utm_content,
+    ua_version,
+    ua_browser,
+    entrypoint,
   FROM
-    firefox_accounts.fxa_content_auth_oauth_events
+    `moz-fx-data-shared-prod.firefox_accounts.fxa_content_auth_oauth_events`
+  WHERE DATE(timestamp) = DATE(@submission_date)  -- # TODO: need to verify how long the flow session lasts. Might have to make this window a bit bigger
```

Comment:
`@submission_date` should already be a `DATE`:
```suggestion
  WHERE DATE(timestamp) = @submission_date  -- # TODO: need to verify how long the flow session lasts. Might have to make this window a bit bigger
```